src/apps/dashboard/features/activity/components/ActivityListItem.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useMemo } from 'react';
+import type { ActivityLogEntry } from '@jellyfin/sdk/lib/generated-client/models/activity-log-entry';
+import Notifications from '@mui/icons-material/Notifications';
+import Avatar from '@mui/material/Avatar';
+import ListItem from '@mui/material/ListItem';
+import ListItemAvatar from '@mui/material/ListItemAvatar';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemText from '@mui/material/ListItemText';
+import Typography from '@mui/material/Typography';
+import formatRelative from 'date-fns/formatRelative';
+import { getLocale } from 'utils/dateFnsLocale';
+import Stack from '@mui/material/Stack';
+
+type IProps = {
+    item: ActivityLogEntry;
+    displayShortOverview: boolean;
+};
+
+const ActivityListItem = ({ item, displayShortOverview }: IProps) => {
+    const relativeDate = useMemo(() => {
+        if (item.Date) {
+            return formatRelative(Date.parse(item.Date), Date.now(), { locale: getLocale() });
+        } else {
+            return 'N/A';
+        }
+    }, [ item ]);
+
+    return (
+        <ListItem disablePadding>
+            <ListItemButton>
+                <ListItemAvatar>
+                    <Avatar sx={{ bgcolor: 'primary.main' }}>
```

Comment:
The color should be based on the "level".

https://github.com/jellyfin/jellyfin-web/blob/master/src/apps/dashboard/features/activity/components/LogLevelChip.tsx#L10-L19